src/types/publication.types.ts

```diff
@@ -104,12 +117,12 @@ interface PublicationEntityDescription {
       volume: string;
```

Comment:
objektet her kan erstattes med `PublicationInstance `

Reply:
trodde jeg gjorde det 😅

Reply:
That's whats QA's for <3